client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -120,10 +136,18 @@ let InviteAccept = React.createClass( {
 	},
 
 	render() {
-		const classes = classNames( 'invite-accept', { 'is-error': !! this.isErrorState() } );
+		const classes = classNames( 'invite-accept', { 'is-error': !! this.isErrorState() } ),
+			{ invite, matchEmailError } = this.state;
 		return (
 			<div className={ classes }>
-				{ ! this.isErrorState() && <InviteHeader { ...this.state.invite } /> }
+				{ matchEmailError &&
```

Comment:
Minor, but can we change `matchEmailError` to something like `forceMatchingEmail`? I suggest that because we are using `isErrorState()` as well, and reading both of those could be confusing at first.